takeout_helper.py

```diff
@@ -277,6 +282,24 @@ def copy_to_target(dir, file):
     return True
 
 
+def copy_to_target_and_divide(dir, file):
+    if not (is_photo(file) or is_video(file)):
+        return True
+
+    try:
+        exif_dict = piexif.load(file)
+    except (piexif.InvalidImageDataError, ValueError):
+        print(f"Couldn't get date for {file}")
+    # creation date like 2019:01:01 23:59:59
+    creation_date = exif_dict['0th'][TAG_DATE_TIME]
+    date = datetime.strptime(creation_date.split()[0], "%Y:%m:%d")
```

Comment:
Getting date from exif is not a good idea, since some files don't have it, some don't support it. My script sets file modification date to this from json/exif/folder's name - so file's modification date is more reliable, use that.

Reply:
So if I understand correctly you set the modification date of the photos/videos to be the date they were taken in?

Reply:
Yes. So that everything incompatible with exif data will still have correct date.

Reply:
For example, videos. *Some* of them have exif, but I didn't found any good library for their exif tho :/ If you would have some spare time, you could also check that :)

Reply:
Got it, it should be an easy change. 
Also like I said, I don't have a way to test it since I don't have pics from google. I guess doing on a random folder with photos should work though, right?

Reply:
Oh, in that case, I'll test it. Good for you that you don't have your pics in Google :)
Tree of takeout looks something like this:
```
Takeout/Zdjęcia Google/
├── 2013-08-06
│   └── metadane.json
├── 2015-06-09
│   └── metadane.json
├── 2016-05-24
│   ├── 20160523_160519_HDR.jpg
│   ├── 20160523_160519_HDR.jpg.json
│   └── metadane.json
```
So you have these "date folders" as I call them with photos and json data for them. If you give it just photos without json, it will figure out dates from exif/folder name.

Reply:
That should do it for getting dates based on modified date. Please let me know of any errors.
About exif, I remember I had a hard time when making the original script to the point where I decided to only support photos since I was making it for me and didn't have many videos anyway